File: programacion/2_Estructuras_de_control/ejercicios_clase/2_1_Sentencias_condicionales/ejercicio_2_1_2.py
# Escribir un programa que almacene la cadena de caracteres contraseña 
# en una variable, pregunte al usuario por la contraseña e imprima 
# por pantalla si la contraseña introducida por el usuario coincide 
# con la guardada en la variable sin tener en cuenta mayúsculas y minúsculas.

password = "Contraseña"

pregunta = input("Ingrese la contraseña: ")

if password == pregunta :
    print("la contraseña introducida es correcta. ")
else :
    print("la contraseña introducida no es correcta. ")

File: programacion/4_POO/0_KotlinBasico_EstructurasDeControl/src/Ejercicio212.kt
/*
Escribir un programa que almacene la cadena de caracteres contraseña en una
variable, pregunte al usuario por la contraseña e imprima por pantalla si la contraseña
introducida por el usuario coincide con la guardada en la variable sin tener en cuenta
mayúsculas y minúsculas
*/

fun main{
    val passwordGuardada: String = "alberti"

    print ("introduce una contraseña: ")
    var passwordUsuario = readLine().lowercase()

    if(passwordUsuario != null) && passwordUsuario.equals(passwordGuardada){
        println("La contraseña es CORRECTA")
    }else{
        println("La contraseña es incorrecta")
    }
}